docs/advanced/document-store/index.md

```diff
@@ -564,6 +564,43 @@ The solution is simple: use {ref}`column-selector<bulk-access>`:
 da[0, 'text'] = 'hello'
 ```
 
+### Offset2id is slow
+DocArray allows list-like behavior by adding offset2id structure to storage backends. This means the database stores, 
+along with documents, meta information about document order.
```

Comment:
This needs to be reworded, but I'm not sure how because the sentence beginning with "This means…" is unclear to me. It seems like there's a verb missing, perhaps. Can you clarify the meaning of that sentence?

Reply:
Updated the document here.